### ISSUE TITLE
Change facet label from "SW Genre" to "MODS Genre"

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/argo#642]
Per @shelleydoljack 

"The SW Genre facet only has 4 terms in it: Tech Reports, Gov Docs, Conference Proceedings, Thesis/dissertation. What displays in the SW Genre facet in Argo is really “MODS Genre”. I think changing the label would be helpful so users aren’t confused why genre terms in their collection display in Argo but not in SearchWorks."

**Comment by blalbrit** [comment]
#644

**Comment by ndushay** [comment]
Note that this facet would get the same values as SW genre if we unpinned the stanford-mods gem in dor-services:

```
Gem::Specification.new do |s|
  s.add_dependency 'stanford-mods', '= 1.5.3'  # v1.5.4 changes what sw_genre returns, which breaks test expectations.
  # TODO: (stanford-mods) figure out how to keep argo indexed counterparts of searchworks fields in sync w/ current SW behavior.
```

**Comment by LynnMcRae** [comment]
oooo, so we had the right name and implementation just an older version of the Genre logic?  We'd prefer the SW version as originally intended and will deal with the reindexing needs.

**Comment by ndushay** [comment]
right - the SW implementation changed;  the stanford-mods gem versioning should have changed from 1.5.3  to   2.0.0   but we messed up semantic versioning.

**Comment by ndushay** [comment]
If you really just want the label changed, it will take me 5 min to do a pull request, and it can be deployed as soon as next argo code is deployed.

**Comment by LynnMcRae** [comment]
Just changing the label is dangerous because it would become wrong as soon as the Stanford MODS Gem advanced.  The SW Genre was the desired facet from the start. The label change was just a quick repair in order not to mislead not knowing this other information.

**Comment by ndushay** [comment]
@blalbrit  @LynnMcRae so should this be closed in favor of #644 ?    What actions should happen and what should their priorities be?

**Comment by LynnMcRae** [comment]
We will not do this since we discovered we were only delaying the cutover to the new set of values values for SW Genre by pinning the version of stanford-mods used by Argo indexing pending discussion.  See https://github.com/sul-dlss/argo/issues/644 instead.
